tasks/network_tracer.py

```diff
@@ -44,7 +44,7 @@ def build(ctx, race=False, rebuild_ebpf_builder=False, incremental_build=False,
 
     # Add custom ld flags
     ldflags += ' '.join(["-X '{name}={value}'".format(name=main+key, value=value) for key, value in ld_vars.items()])
-    build_tags = get_default_build_tags(puppy=puppy).append(BPF_TAG)
+    build_tags = get_default_build_tags(puppy=puppy) + [BPF_TAG]
 
     # TODO static option
```

Comment:
Coming in another PR? Possibly being left out if we don't need static builds?

Reply:
Mostly left out if we don't need static builds, if we do we could just import this part too: https://github.com/DataDog/datadog-process-agent/blob/master/gorake.rb#L58-L72